pkg/authprovider/authx/testData/example-auth.enc.yaml

```
@@ -0,0 +1,89 @@
+            cixmk997cakhJVevjlE2tmdt93WUR8Z4TY+xFMgQf5bz9v8vJG3fpgGxoqZ3pIsW
+            LjMLaoTVO9gammmml15gaJdV9PolpbwSHMgFvoVv4eSADsCO1Cs1zFrFbTbmHq2L
+            4wUa6f9Df6jS4M4radl5mh+AQI2JCvwTia1eVduDZxTiKxXDGTwYIDDsLd8GZrf/
+            Vfy/bwhmNtSCFx9raavSjCURpfNSqhkQnMnrsTpIVe8bl+YQNkmJY4+ZUGFeVXYH
+            GzQm7AIxTsUe5dYO3T1W0lwBfxopAXTorhFy9VsMY7ZjaOUPnOrFyCLD8+WXUERU
+            B/ixpZ2X+0Qeax5vnAaykJKr9UNVW5oxS4mfPvQ8+AUUsjJdC3RZHOKIlHTbvT0v
+            aXOSeHkaNJ+TWnRBow==
+            =tiYG
+            -----END PGP MESSAGE-----
+          fp: C6C11FC4DA74DC979109CBD48EB7405A1F8D1930
+    encrypted_regex: ^(password|username|token|value|key|raw)$
+    version: 3.9.1
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)